src/tracer.js

```diff
@@ -177,10 +189,9 @@ function sendCurrentTrace(traceSender) {
 function postTrace(traceObject) {
     utils.debugLog(`Posting trace to ${config.getConfig().traceCollectorURL}`);
     utils.debugLog(`trace: ${JSON.stringify(traceObject, null, 2)}`);
-    return axios.post(
+    return module.exports.sessionPost(
```

Comment:
is there no way the connection will be closed? (a keep alive might not be sent because the container did not get enough runtime?) maybe we need to check and re-open if needed?

Reply:
tested the following scenarios:
1. HTTP server (i.e. TC) sends a Connection: closed header.
2. socket.close event (fin).

In both case it works OK